codegen/settings.gradle.kts

```diff
@@ -5,9 +5,6 @@ rootProject.name = "smithy-dafny"
 include(":smithy-dafny-codegen")
 include(":smithy-dafny-codegen-cli")
 include(":smithy-dafny-codegen-test")
-// TODO: Once Smithy-Python is published to Maven, and we do not rely on a fork, use that
```

Comment:
This should probably be tracked in an issue for both Python and Rust

Reply:
You mean an issue in Smithy-Python/rs that says "Publish codegen to Maven"? I agree

Reply:
Python: https://github.com/smithy-lang/smithy-python/issues/296
Rust already had one: https://github.com/smithy-lang/smithy-rs/issues/421